peers/src/consensus_msg.rs

```diff
@@ -29,6 +29,9 @@ pub enum ConsensusValue {
 
     /// MintTx({0})
     MintTx(MintTx),
+
+    /// Timestamp({0})ms
+    TimeStamp(u64),
```

Comment:
Super nit but `TimeStamp` or `Timestamp`? I don't know if I've encountered it as two separate words

Reply:
fixed in 3733